src/exports.jl

```diff
@@ -247,6 +248,7 @@ export
   lognorm,
   movesite,
   movesites,
+  ndims,
```

Comment:
Seems strange to export this since it is a `Base` function.

Reply:
I'll try removing it.